client/state/sites/test/selectors.js

```diff
@@ -1184,4 +1194,675 @@ describe( 'selectors', () => {
+								admin_url: 'https://jetpacksite.me/wp-admin/',
+								jetpack_version: '3.3'
+							}
+						}
+					}
+				}
+			};
+
+			const managementUrl = getJetpackSiteRemoteManagementURL( state, siteId );
+			expect( managementUrl ).to.equal( 'https://jetpacksite.me/wp-admin/admin.php?page=jetpack&configure=manage' );
+		} );
+
+		it( 'it should return the correct url for versions of jetpack greater than or equal to 3.4', () => {
+			const siteId = 77203074;
+
+			const state = {
+				sites: {
+					items: {
+						77203074: {
+							ID: siteId,
+							URL: 'https://jetpacksite.me',
+							jetpack: true,
+							options: {
+								active_modules: [ 'manage', 'sso', 'photon', 'omnisearch' ],
+								admin_url: 'https://jetpacksite.me/wp-admin/',
+								jetpack_version: '3.4'
+							}
+						}
+					}
+				}
+			};
+
+			const managementUrl = getJetpackSiteRemoteManagementURL( state, siteId );
+			expect( managementUrl ).to.equal( 'https://jetpacksite.me/wp-admin/admin.php?page=jetpack&configure=json-api' );
+		} );
+	} );
+
+	describe( '#hasJetpackSiteCustomDomain()', () => {
+		it( 'should return `null` for a non-existing site', () => {
+			const state = {
+				sites: {
+					items: {}
+				}
+			};
+			let siteId;
+
+			const hasCustomDomain = hasJetpackSiteCustomDomain( state, siteId );
+			expect( hasCustomDomain ).to.equal( null );
+		} );
+
+		it( 'it should return `true` if `URL` and `unmapped_url` have the same domain', () => {
+			const siteId = 77203074;
+
+			const state = {
+				sites: {
+					items: {
+						77203074: {
+							ID: siteId,
+							URL: 'https://jetpacksite.me',
+							options: {
+								unmapped_url: 'https://jetpack.co'
+							}
+						}
+					}
+				}
+			};
+
+			const hasCustomDomain = hasJetpackSiteCustomDomain( state, siteId );
+			expect( hasCustomDomain ).to.equal( true );
+		} );
+
+		it( 'it should return `false` if `URL` and `unmapped_url` have different domains', () => {
+			const siteId = 77203074;
+
+			const state = {
+				sites: {
+					items: {
+						77203074: {
+							ID: siteId,
+							URL: 'https://jetpacksite.me',
+							options: {
+								unmapped_url: 'https://jetpacksite.me'
+							}
+						}
+					}
+				}
+			};
+
+			const hasCustomDomain = hasJetpackSiteCustomDomain( state, siteId );
+			expect( hasCustomDomain ).to.equal( false );
+		} );
+	} );
+
+	describe( '#getJetpackSiteFileModDisableReasons()', () => {
+		it( 'it should have the correct reason for the clue `has_no_file_system_write_access`', () => {
+			const siteId = 77203074;
+
+			const state = {
+				sites: {
+					items: {
+						77203074: {
+							ID: siteId,
+							options: {
+								file_mod_disabled: [ 'has_no_file_system_write_access' ]
+							}
+						}
+					}
+				}
+			};
+
+			const reason = getJetpackSiteFileModDisableReasons( state, siteId );
+			expect( reason ).to.deep.equal( [ i18n.translate( 'The file permissions on this host prevent editing files.' ) ] );
```

Comment:
I think you can skip `translate` bits in tests. We always use english version in tests.